tests/integration/s2n_test_constants.py

```diff
@@ -37,7 +37,7 @@ class OCSP(Enum):
 
 # Specifying a single cipher suite in GnuTLS requires specifying a "priority string" that removes all cipher suites,
 # and then adds each algorithm(kx,auth,enc,mac) for a given suite. See https://www.gnutls.org/manual/html_node/Priority-Strings.html
-S2N_GNUTLS_PRIORITY_PREFIX="NONE:+COMP-NULL:+CTYPE-ALL:+CURVE-ALL"
+S2N_GNUTLS_PRIORITY_PREFIX="NONE:+COMP-NULL:+CTYPE-ALL"
```

Comment:
why are you removing the CURVE-ALL here?

Reply:
We need to remove it from here for adding curve test, and CURVE-ALL is moved to other gnutls tests.